### DISCUSSION TITLE
Copy not working

### DISCUSSION THREAD

**Discussion by mohamed-umar** [nick-keller/react-datasheet-grid#114] [Q&A] [answered]
Hey!! Your Library is awesome to work with.
But just faced an issue when I tried to copy data from sheet.

<img width="457" alt="Screenshot 2022-01-27 at 11 45 50 AM" src="https://user-images.githubusercontent.com/59644436/151302415-f6b86b84-4f16-4e30-bf1c-65efce1180e1.png">
<img width="284" alt="Screenshot 2022-01-27 at 11 47 12 AM" src="https://user-images.githubusercontent.com/59644436/151302580-84c18e47-a3ba-468a-afc2-a0fabf1bb086.png">

Can you help me on this.???

**Comment by nick-keller** [comment, marked as answer]
Hello @mohamed-umar ,
sure, can you show the code that would allow me to reproduce the error? Looking at the unit tests and the demo app it looks like it's working so I need to look at your particular use case.

**Reply by mohamed-umar** [reply]
This is how I used it...
<img width="666" alt="Screenshot 2022-01-27 at 3 10 17 PM" src="https://user-images.githubusercontent.com/59644436/151332814-764117d7-9bf8-41bf-b225-f16009d81897.png">

**Reply by mohamed-umar** [reply]
@nick-keller Also Just went through Copy event code , Where you are using an enodeHtml() from "html-entities@2.3.2" but locally i have older html-entities version... Will that be any cause for my issue..

**Reply by nick-keller** [reply]
That sounds like it is the issue, try updating the lib to a newer version
